src/domains/ecommerce-app/README.md

```diff
@@ -1,5 +1,5 @@
 <!-- markdownlint-disable -->
-<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+<!-- BEGIN_TF_DOCS -->
```

Comment:
maybe here we have to rollback those pre-commit changes?

Reply:
Why should we rollback?